Tella/Scenes/Settings/Models/DeleteServerTexts.swift

```diff
@@ -9,18 +9,58 @@
 import Foundation
 
 // TODO: Maybe use another class name
-struct DeleteServerTexts {
-    var titleText: String = ""
-    var messageText: String = ""
-    let cancelText = "CANCEL"
-    let actionText = "DELETE"
+enum DeleteServerTexts {
+    case tella(String)
+    case uwazi(String)
+    case gDrive(String)
+    case unknown
+
+    var titleText: String {
+        switch self {
+        case .tella(let name):
+            return "Delete \(name) server?"
+        case .uwazi(let name):
+            return "Delete \"\(name)\" connection?"
+        case .gDrive(let name):
+            return "Delete \"\(name)\" connection?"
+        case .unknown:
+            return ""
+        }
+    }
+    
+    var messageText: String {
+        switch self {
+        case .tella:
+            return "if you delete this server, all draft and submitted forms will be deleted from your device."
+        case .uwazi:
+            return "If you delete this server, all draft and submitted entities will be deleted from your device. Delete anyway?"
+        case .gDrive:
+            return "If you delete this server, all draft and submitted reports will be deleted from your device."
+        case .unknown:
+            return ""
```

Comment:
Can we use the same messageText texts look the same for all the servertypes!

Reply:
I added localization for this but I will leave a different message for Uwazi just for now to follow exactly the figma designs